core/src/OpenMP/Kokkos_OpenMP_Instance.cpp

```diff
@@ -57,19 +54,20 @@
 #include <impl/Kokkos_Error.hpp>
 #include <impl/Kokkos_CPUDiscovery.hpp>
 #include <impl/Kokkos_Tools.hpp>
+#include "Kokkos_OpenMP_Instance.hpp"
```

Comment:
Don't we get it transitively from `<Kokkos_OpenMP.hpp>` which is included from `<Kokkos_Core.hpp>`
(I don't necessary have an issue with your explicitly including it)

Reply:
Yes, you do. It's just the habit of adding the `.hpp` with the corresponding `.cpp`.